aws_batch/indexing.py

```diff
@@ -25,20 +27,62 @@ def execute_command(command):
     output = subprocess.check_output(command, shell=True)
     return output
 
+def split_fasta(fasta_file, max_fasta_part_size):
+    fasta_file_list = []
+    part_idx = 0; current_size = 0
+    current_output_file_name = "%s.%d" % (fasta_file, part_idx)
+    current_output_file = open(current_output_file_name, 'wb')
+    fasta_file_list.append(current_output_file_name)
+    with open(fasta_file, 'rb') as input_f:
+        current_read = input_f.readline()
+        for line in input_f:
```

Comment:
i believe in python 2.7 this "for line in input_f" reads the entire file in memory.

Reply:
https://stackoverflow.com/questions/8009882/how-to-read-large-file-line-by-line-in-python

Reply:
there is a small probability the last fragment would be very short

Reply:
yeah i just found that stackoverflow question myself, the answer says this is good for python versions >= 2.6

i think if this doesn't blow up for our 4GB file we are good :)

Reply:
yeah. i tested the indexing part. it works.